src/_posts/databases/2000-01-01-maintenance-windows.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Database Maintenance Windows
+nav: Maintenance Windows
+modified_at: 2024-11-28 12:00:00
+tags: databases maintenance
+index: 5
+---
+
+
+## Understanding Database Maintenance Windows
+
+**Database Maintenance Windows** is Scalingo's system for scheduling
+*[maintenance operations](#understanding-maintenance-operations)* on your
+databases. On the customer's side, it mainly consists in an 8 hours timespan
+during which database maintenance operations can be scheduled. The goal of
+maintenance windows is to minimize the impact of maintenance operations on your
+application, while facilitating continuous improvement on our side.
+
+{% note %}
+Currently, Scalingo only leverages maintenance windows to force
+[major-upgrades]({% post_url databases/postgresql/2000-01-01-upgrading %})
+of databases when very specific conditions are met, such as an EOL version
+still running, or critical security issues.
+{% endnote %}
+
+A default 8 hours maintenance window is automatically assigned to all newly
+provisioned database. This maintenance window is set to a random weekday,
+from 9PM to 5AM UTC.
+
+If the given timeframe doesn't align with your business requirements, please
+[adjust it as needed](#configuring-a-database-maintenance-window). We strongly
+advise to schedule the maintenance window when your app is less prone to high
+traffic (e.g. the night between Sunday and Monday) to minimize undesired
+consequences such as a possible downtime.
+
+{% note %}
+Database Maintenance Windows are excluded from the SLA calculation.
+{% endnote %}
+
+
+## Viewing Database Maintenance Windows
+
+### Using the Database Dashboard
+
+1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
```

Comment:
question: this link targets a PostgreSQL-specific page whereas we are on a page that is more general for all database types. Doesn't it feel odd?

Reply:
Indeed, I removed the links.
Thanks <3